CIScripts/Test/TestRunner.ps1

```diff
@@ -103,3 +103,36 @@ function Run-Tests {
         }
     })
 }
+
+function Run-Tests {
+    Param ([Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions)
+
+    try {
+        Run-TestsWithoutLogs -Sessions $Sessions
```

Comment:
Pedantic, but I really don't like this function name. It kinda implicitly makes you think that there's a `Run-TestsWithLogs` function or that the tests that are being ran don't produce logs.
I think maybe something like this would be better:
```
function Run-Tests{
.. 
   try {
      Run-TestScenarios -Sessions $sessions
   } catch {
      Collect-Logs -Sessions $sessions
   }
}

function Run-TestScenarios {
   // current body of Run-Tests
}

function Collect-Logs {
   // what you put in "catch" block
}
```

Reply:
I wasn't happy with naming either. I like your suggested version!
@PawelMarchewkaCodiLime The reason I didn't just add `try` to `Run-Tests` was to avoid unnecessary indentation and make the diff cleaner.